styles/main.css

```diff
@@ -92,14 +101,16 @@ footer{
 }
 
 footer hr{
-	width:50%;
+	width:50vw;
```

Comment:
vw is the unit to use if you want to specify a size relative to the browser viewport.  Use % to specify a size relative to the element's parent